src/custom/pages/Claim/styled.ts

```diff
@@ -188,8 +191,10 @@ export const TokenLogo = styled.div<{ symbol: string; size: number }>`
   width: ${({ size }) => `${size}px`};
   height: ${({ size }) => `${size}px`};
   border-radius: ${({ size }) => `${size}px`};
-  /* background: ${({ symbol }) => `url(${symbol}.png) no-repeat center/contain`}; */
-  background: grey;
+  background: ${({ symbol, theme }) =>
+    `url(${
+      symbol === 'GNO' ? LogoGNO : symbol === 'ETH' ? LogoETH : symbol === 'USDC' ? LogoUSDC : theme.blueShade3
```

Comment:
Function would likely be cleaner, but given this limited scope of 3 token symbols......

Reply:
Still don't know how to simplify this where I can use the prop `symbol` as the variable name instead. So that we could do: `url(${symbol}.png`. 

Perhaps like `props[symbol]` => `'GNO'` ?

Reply:
Just make a map that has token symbol as keys and logo as values

```ts
const TokenSymbolToLogoMap = {
  GNO: GNOLogo,
  // ...rest
}
```
Ideally we have some enum of the claim tokens you can use as dynamic keys